File: Programmers/level1/controlStrings/control_strings.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12918

def solution(s):
    if len(s) == 4 or len(s) == 6:
        if s.isdigit():
            return True
        else:
            return False
    else:
        return False

File: Programmers/level1/controlStrings/controlStrings.js
// https://programmers.co.kr/learn/courses/30/lessons/12918?language=javascript

export const solution = (s) => {
    if(s.length === 4 || s.length === 6){
        if(s.includes('e')) return false;
        if(Number(s)) return true;
        return false;
    } else {
        return false;
    }
}

// Number 함수는 e도 숫자로 변환하여 지수로 계산